tools/openms/macros.xml

```diff
@@ -4,7 +4,7 @@
      add additional macro files using the m/macros parameter -->
 <macros>
   <token name="@TOOL_VERSION@">2.6</token>
-  <token name="@GALAXY_VERSION@">0</token>
+  <token name="@GALAXY_VERSION@">1</token>
```

Comment:
can we just bump the tools that need to be bumped and not all 120 tools. I'm really worried about our toolbox and the parsing/loading/caching involved.

Reply:
Sure. But I also have no feedback yet from the OpenMS community if they prefer to continue with updated tools or to collect more issues which I could solve in this PR. Will ask again.

Reply:
Just asking myself how to manage this with automatic tool building. One would need to remove the central `@GALAXY_VERSION@` and bump only tools that really change, i.e. compare new and old xml.